### PR TITLE
Testing/forgot password

### PR DESCRIPTION
**Pull request by gibrandarmawan**
Added testing suite for forgot password and README.md

### CODE REVIEW

File: login_tests/resource.robot [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+...               by the imported SeleniumLibrary.
+Library           SeleniumLibrary
+Library           Dialogs
+
+*** Variables ***
+${SERVER}         login-dxt360.sonarplatform.com
+${BROWSER}        Chrome
+${DELAY}          0
+${VALID USER}     demo.analytics@dxt360.app
+${VALID PASSWORD}    p3iN%u49
```
Comment (cendekia) [inline login_tests/resource.robot:15]: ini harus pakai environment variable ya @gibrandarmawan

**Review by cendekia — CHANGES_REQUESTED** [review]

File: login_tests/resource.robot (hunk 1 above)
Comment (gibrandarmawan) [reply login_tests/resource.robot:15]: bisa diganti jadi argument via terminal mas

Comment (cendekia) [reply login_tests/resource.robot:15]: install dotenv, then add file `.env` and put all sensitive parameters in it, make .env file ignored by git

then add also `.env.example` file to give what are variable the used by application so other devs will know it

Comment (cendekia) [reply login_tests/resource.robot:15]: intinya sensitive values should not shared in git.

Comment (gibrandarmawan) [reply login_tests/resource.robot:15]: oke mas, saya ganti dulu

**Pull request merged**